addons/mail/static/src/utils/common/format.js

```diff
@@ -35,32 +35,58 @@ const _escapeEntities = (function () {
     };
 })();
 
+const _escape = function (str) {
```

Comment:
Is this a bad rebase? This should not be re-defind here

Reply:
it is used for the escape for the emoji generate, not sure if i can find a better escape that exclude the " ' `